Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
When I ran this, I noticed that the cursor passed right through the UI. The buttons lit up and could be tapped, but I found it visually disturbing that the cursor remained far behind the buttons instead of snapping to a collider on the buttons.